### DISCUSSION TITLE
mitmproxy 8.1.1 is identified as malware by McAfee

### DISCUSSION THREAD

**Discussion by CruzMarcio** [mitmproxy/mitmproxy#5467] [General]
McAfee Endpoint Security identifies mitmproxy.exe as malware:

Adaptive Threat Protection repaired C:\tools\mitmproxy\mitmweb.exe TargetType, because its reputation (Known Malicious) is below the configured Clean threshold.

Analyzer / Detector
--
Product name | McAfee Endpoint Security
Product version | 10.7.0.3183
Feature name | Real Protect Cloud
 
Threat
Action taken | Clean
Threat category | Malware Detected
Threat event ID | 35107
Threat handled | Yes
Threat name | Real Protect-peft7!4CF8F35B22D3
Threat severity | Critical
Threat timestamp | 7/19/2022 9:08 AM
Threat type | Trojan
 
Target
Target hash | 4cf8f35b22d371a1bebfa4bf4c5b03ee
Target name | mitmweb.exe

**Comment by mhils** [comment]
Same as https://github.com/mitmproxy/mitmproxy/discussions/5431#discussioncomment-3033110. Would you mind submitting it as a false-positive to McAfee please?

**Comment by virtualwatt** [comment]
I have the same problem. Our company security department reports that the hash is recognized as malware by several AV vendors beside McAfee, please change it!
[Detection screenshot](https://1drv.ms/u/s!AhDwWd7wOGnLon4OSAOzToolcWbO)

**Reply by mhils** [reply]
Somewhat related, this issue would likely be fixed by installing from the Microsoft Store, see https://github.com/mitmproxy/mitmproxy/discussions/5477. If you have any thoughts on this, please chime in!

**Reply by virtualwatt** [reply]
Oh thanks, there is 8.1.2 in the MS Store and it's fine (choco installs 8.1.1).

**Reply by mhils** [reply]
We're currently testing the store setup, which causes some version number inconsistencies. There formally is no 8.1.2 release, I just needed to bump the number during testing. The store will soon have 8.1.3, which is exactly the same as 8.1.1 everywhere else. With the next release they'll stay in sync.
